doc/src/vtr/server_mode/index.rst

```diff
@@ -0,0 +1,203 @@
+          count_adder_carry_p_cout[14].sumout[0] (adder_carry)                       0.040     3.025
+          count_dffsre_Q_D[13].in[0] (.names)                                        0.564     3.589
+          count_dffsre_Q_D[13].out[0] (.names)                                       0.228     3.818
+          count[13].D[0] (dffsre)                                                    0.000     3.818
+          data arrival time                                                                    3.818
+
+          clock clk (rise edge)                                                      0.000     0.000
+          clock source latency                                                       0.000     0.000
+          clk.inpad[0] (.input)                                                      0.000     0.000
+          count[13].C[0] (dffsre)                                                    0.715     0.715
+          clock uncertainty                                                          0.000     0.715
+          cell setup time                                                           -0.057     0.659
+          data required time                                                                   0.659
+          ------------------------------------------------------------------------------------------
+          data required time                                                                   0.659
+          data arrival time                                                                   -3.818
+          ------------------------------------------------------------------------------------------
+          slack (VIOLATED)                                                                    -3.159
+
+
+          #End of timing report
+          #RPT METADATA:
+          path_index/clock_launch_path_elements_num/arrival_path_elements_num
+          0/2/30
+      ",
+        "STATUS": "1"
+      }
+
+Draw selected critical path elements example
+--------------------------------------------
+
+  Let's take a look at an example of a request timing report telegram body with the following options:
+
+  - path_elements = path 0 and it's sub-elements 10,11,12,13,14,15,20,21,22,23,24,25
+  - high_light_mode = "crit path flylines delays"
+  - draw_path_contour = 1
+
+  .. code-block:: json
+      :caption: telegram body **REQUEST** example
+      :linenos:
+
+      {
+        "CMD": "1",
+        "JOB_ID": "2",
+        "OPTIONS": "string:path_elements:0#10,11,12,13,14,15,20,21,22,23,24,25;string:high_light_mode:crit path flylines delays;bool:draw_path_contour:1"
+      }
+
+  **high_light_mode** could have following string values:
+
+  - "crit path flylines"
+  - "crit path flylines delays"
+  - "crit path routing"
+  - "crit path routing delays"
+
+  Response will look like:
+
+  .. code-block:: json
+      :caption: telegram body **RESPONSE** example
+      :linenos:
+
+      {
+        "JOB_ID": "2",
+        "CMD": "1",
```

Comment:
Again, not sure order matters (not sure if you're showing the raw data over the socket or just showing what a valid json parsing of it would be), but CMD and JOB_ID are the opposite order in the figure.

Reply:
The order of elements in telegram body is not matter, since it's json format.
The order of bytes is matter only for telegram header, which is fixed size structure.
Anyway i will update order of fields in json to be match.

Reply:
solved https://github.com/w0lek/vtr-verilog-to-routing/pull/11/commits/05ecf74d8b2781788237df5ea16f52e8b6a3645a
(will be updated later)